uiowa/uiowa
# Media Tagging Support

Resolves https://github.com/uiowa/uiowa/issues/137

This satisfies the first step towards something more https://github.com/uiowa/uiowa/issues/3453. Site editors can tag media and find media by tag. Tagged media _do not_ show on taxonomy term pages but _are_ visible in the usage tab. This PR is affected by... you guessed it... https://github.com/uiowa/uiowa/issues/3204

# Story

As a site editor, I can tag new and existing media entities for organizational purposes and then filter by tag on the media overview and media library/browser displays.

# To Test

- sync admissions, v2, v3 sites.
- When adding or editing media (all default types), you can add tags to the media entity
- You can filter by those tags within the media overview page or when adding media via wysiwyg
- When you view a taxonomy term page, the referenced media entities should not display similar to tagged content (page, article, person) but can be seen within the usage tab of the taxonomy term.
- Admissions does not have any config import issues.
